ansible_base/feature_flags/flag_source.py

```diff
@@ -0,0 +1,25 @@
+from django.apps import apps
+from flags.sources import Condition
+
+
+class DatabaseCondition(Condition):
+    """Condition that includes the AAPFlags database object"""
```

Comment:
I'm going to have to admit I can't understand this class and its docstring easily. It would make sense if I could then connect it to what the init method is doing below. But that's very non-obvious.

Reply:
Good note, I will update this docstring to better define it.

Reply:
After checking further, this class isn't needed and I removed it.

Reply:
I realize now this function was required to ensure that the django-flag functions `enable_flag/disable_flag` can work as intended.
I added this back, clarified the doc string, and added a test to cover this case.